mod_reforged/hooks/ai/tactical/behaviors/ai_attack_throw_net.nut

```diff
@@ -4,6 +4,18 @@
 	q.onEvaluate = @(__original) function( _entity )
 	{
 		local ret = __original(_entity);
+		// It is a generator in vanilla, but we add the typeof check for maximum
+		// compatibility if another mod changed the original function to not be a generator.
+		if (typeof ret == "generator")
+		{
+			local thread = ret;
+			do
+			{
+				ret = resume thread;
```

Comment:
You do not call any `yield null` which means onEvaluate is now no longer a generator.
I think that will cause a runtime error in vanilla, because vanilla tries to call `resume` on the first value which is returned by `onEvaluate`